pandas/core/frame.py

```diff
@@ -5444,6 +5444,34 @@ def shift(
     ) -> DataFrame:
         axis = self._get_axis_number(axis)
 
+        # Handle the case of multiple shifts
```

Comment:
prob reasonable to add a suffix arg (and if None use this as a default)

Reply:
Thank you for the suggestion, I will add that in!

Reply:
We added in the agument

`suffix=None`

and in our shift:

`.add_suffix(f"_{i}" if suffix is None else suffix) `